### ISSUE TITLE
Remove/replace SideShift

### ISSUE THREAD

**Issue by Forage** [m2049r/xmrwallet#919]
SideShift.ai doesn't support Monero any longer, so the service can no longer be used in the wallet. It doesn't appear to be temporary either: https://github.com/monero-project/monero-site/pull/2183

**Comment by d4f5409d** [comment]
https://fixedfloat.com would be a good replacement

(StealthEX has 1 USD fee)

**Comment by xmready** [comment]
What's the status on this issue/feature?

**Comment by d4f5409d** [comment]
Afaik, Sideshift still works from the app but not on the website front, but I am not sure since I didn't risk losing my funds

**Comment by xmready** [comment]
> Afaik, Sideshift still works from the app but not on the website front, but I am not sure since I didn't risk losing my funds

Last time I tried to use sideshift through the app it failed over and over again. I emailed the Monerujo dev and they said sideshift would be replaced soon. This was a while ago now. No word. @m2049r

**Comment by d4f5409d** [comment]
It'd be good time for some clarification because this is very shady from both parties.

**Comment by tajo48** [comment]
Still down

**Comment by Leza89** [comment]
I would like to chime in and stress the importance of this feature.
I have installed Monerujo for a friend of mine and set them up with their first millineros so they can go and use them and to get some adoption, hopefully.
We also came across the issue that Monero is not widely used (even Bitcoin, sadly, still is not really accepted, but paying in Monero is much harder), so I specifically told my friend that they can pay in Bitcoin as well with Monerujo. Today I tried to donate in BTC myself and I had to experience that it is not working.

+1 for the replacement
+1 for the idea of having multiple services (ideally with a selection of the best exchange fee + mining fee combination)

**Comment by d4f5409d** [comment]
The dev doesn't care anymore. The project is abandoned. There are barely if any commits.

**Comment by xmready** [comment]
@m2049r we see you making commits, why can't you respond to these issues? Its a bad look to ignore users when your app's purpose is to manage people's money.

**Comment by Leza89** [comment]
Let's not forget that this is _free_ software @xmready 

I'd like to hear though what @m2049r 's opinion on the matter is, at least. Maybe he expects a reactivation of the service?

Best long-term solution would be to be able to add custom payment services, I guess. populate the list initially, but allow for user customization.

**Comment by xmready** [comment]
> Let's not forget that this is _free_ software @xmready
> 
> I'd like to hear though what @m2049r 's opinion on the matter is, at least. Maybe he expects a reactivation of the service?
> 
> Best long-term solution would be to be able to add custom payment services, I guess. populate the list initially, but allow for user customization.

The purpose of this free software is to manage people money. It's not a calendar app or something trivial. XMR solutions need to be transparent and secure in their development with clearly defined goals. 

The dev has stopped responding to emails and issues. This is concerning given the purpose of this software. My intention is to figure out if I need to move to a wallet that has transparent communication and intention with its user base.

**Comment by anhdres** [comment]
Some of the reasons we move this slow and we try very hard to not break things is specially because it's people's money, including my own money, which of course I keep in Monerujo.

Sideshift gave no prior warning about delisting XMR, and it was specially confusing because it delisted it from the website, but not from the API, and it was until months later that it finally stopped working for most people.

As you might have notice, we're either not very profit-driven, or we're pretty bad at it. It has its benefits if you ask me. It's tempting to remove all exchanges altogether but *many* people asked us to not only fix it but add multiple options to it. We're trying to give it our own spin to it, and in true Monerujo fashion, we overengineer stuff.

Bottom line, we've tested and keep testing many services, and have mockups and have tested builds with other options. We'll release it once we feel it works well enough. Sadly, we don't work full time in Monerujo, so it takes more time that we'd like.

That aside, the beauty of having standards is that you can restore your wallet using other apps, and nowadays there are more than one options. I personally believe Monerujo is beautifully unique in some respects and equally frustrating in some others. It's not for everybody, that's for sure.

**Comment by xmready** [comment]
> Some of the reasons we move this slow and we try very hard to not break things is specially because it's people's money, including my own money, which of course I keep in Monerujo.
> 
> Sideshift gave no prior warning about delisting XMR, and it was specially confusing because it delisted it from the website, but not from the API, and it was until months later that it finally stopped working for most people.
> 
> As you might have notice, we're either not very profit-driven, or we're pretty bad at it. It has its benefits if you ask me. It's tempting to remove all exchanges altogether but _many_ people asked us to not only fix it but add multiple options to it. We're trying to give it our own spin to it, and in true Monerujo fashion, we overengineer stuff.
> 
> Bottom line, we've tested and keep testing many services, and have mockups and have tested builds with other options. We'll release it once we feel it works well enough. Sadly, we don't work full time in Monerujo, so it takes more time that we'd like.
> 
> That aside, the beauty of having standards is that you can restore your wallet using other apps, and nowadays there are more than one options. I personally believe Monerujo is beautifully unique in some respects and equally frustrating in some others. It's not for everybody, that's for sure.

Not sure why it took soo long for someone to write a reply like this, but thanks for breaking the silence. Its reassuring to hear from the devs, regardless of how fast things are moving.
